examples/toggle.rs

```diff
@@ -0,0 +1,42 @@
+//! Uses the StatefulOutputPin embedded_hal trait to toggle the pin
+//! On the stm32 discovery board this is the red led
+//! Target board: STM32F3DISCOVERY
+
+#![deny(unsafe_code)]
+#![no_main]
+#![no_std]
+
+use panic_semihosting as _;
```

Comment:
To stay consistent with:
https://github.com/stm32-rs/stm32f3xx-hal/blob/902b56f8f11221cb8ffbf24305d66ecea858bf6c/examples/usb_serial.rs#L6

```suggestion
extern crate panic_semihosting;
```

Reply:
Personally, I am more in favor of the `use panic_semihosting as _;` method, as this is more Edition 2018 idiomatic (?)

However, consistency is more important, and I will happily change it.